docs/medical-api/api-reference/fhir/get-medical-record-summary.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: "Get Medical Record Summary"
+description: "Get the Medical Record Summary in PDF or HTML formats."
+api: "GET /patient/:id/medical-record"
+---
+
+This endpoint returns the URL to download the patient's Medical Record Summary in the PDF or HTML format, if it exists. 
+
+To create a Medical Record Summary, you need to use the [Start Consolidated Data Query](/medical-api/api-reference/fhir/consolidated-data-query-post). 
+
+## Path Params
+
+<ParamField path="id" type="string" required>
+  The ID of the Patient.
+</ParamField>
+
+## Query Params
+
+<ParamField query="conversionType" type="string" required>
+  The doc type to convert to. Either `html` or `pdf`.
+
+  <Info>This parameter is to indicate how the medical record should be rendered.</Info>
+</ParamField>
+
+## Response
+
+Returns a JSON containing the URL for the Medical Record Summary.
+
+<ResponseExample>
+
+```javascript Metriport SDK
+import { MetriportMedicalApi } from "@metriport/api-sdk";
+
+const metriport = new MetriportMedicalApi("YOUR_API_KEY");
+
+const url = await metriport.getMedicalRecord(
+    "018a80c4-292a-7486-a223-6dcbc636c44c", 
+    "pdf"
+);
+```
+
+</ResponseExample>
+
+<ResponseField name="url" type="string">
```

Comment:
see https://docs.metriport.com/medical-api/api-reference/document/get-document for the way done there, since its the same return type. For consistency, should phrase like:
"The downloadable URL" for the url description
Also change the order so the response example is at the end like in get-document.mdx, and include an <Info> tag about duration of MR validaity (180 seconds afaik)